packages/sampler/stories/default/MediaOverlay.js

```diff
@@ -77,6 +77,7 @@ storiesOf('Moonstone', module)
 			const placeholder = prop.placeholder[placeholderName];
 			return (
 				<MediaOverlay
+					disabled={boolean('disabled', Config)}
```

Comment:
I'm not sure `MediaOverlay` is _supposed_ to be disable-able.  So, I'm not sure this makes sense.

Reply:
🤷‍♂ I just did what I was told. I'm not familiar with how MediaOverlay is used.